doc/src/manual/methods.md

```diff
@@ -547,6 +547,8 @@ function getindex(A::AbstractArray{T,N}, indexes::Vararg{Number,N}) where {T,N}
 
 would be called only when the number of `indexes` matches the dimensionality of the array.
 
+When only the type of supplied arguments needs to be constrained `Vararg{T}` can be equivalently written as `T...`. For instance `f(x::Int...) = x` is a shorthand for `f(x::Vararg{Int}) = x`.
```

Comment:
usually recommended to follow the line length convention of the surrounding file